doc/en/changelog.rst

```diff
@@ -28,6 +28,13 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+pytest 6.2.4 (2021-05-04)
```

Comment:
Hmm this doesn't seem right.

Reply:
wait I'm confused, which thing looks wrong

Reply:
oh you fixed it already I see !